docs/source/contributors/workflow.rst

```diff
@@ -94,3 +94,42 @@ the PR that this being backported separately.
 
 If you encounter problems with cherry picking into release branch please
 don't hesitate to speak to an experienced member of the FitBenchmarking team.
+
+
+================
+Create a release
+================
+In order to create a new release for FitBenchmarking, there are a few manual steps.
+These have been streamlined as much as possible.
+
+First checkout the branch to create the release from.
+
+From the root of the repo run the "ci/prep_and_tag_release.sh" script with the new version number.
+The version number will be rejected if it is not of the expected form.
+We expect a "v" followed by the major, minor, and patch numbers,
+and an optional numbered label to mark the type of release.
+
+Possible lables are:
+ - -beta (release for testing)
+ - -rc (release candidate)
+ - -dev (release of a development version)
+
```

Comment:
I know we have these three already, but do we need them?  What's the difference between `dev` and `beta`, for example?

Reply:
I was just listing what we already had... Maybe we'd want a `dev` if someone requested a bleeding edge feature and we weren't near a `beta`? We can probably remove this until we need it.